.nuspec/Xamarin.Forms.nuspec

```diff
@@ -24,6 +24,7 @@
         <dependency id="NETStandard.Library" version="2.0.1"/>
         <dependency id="Microsoft.NETCore.UniversalWindowsPlatform" version="6.0.12" />
         <dependency id="Win2D.uwp" version="1.20.0" />
+        <dependency id="Microsoft.UI.Xaml" version="2.1.190606001" />
```

Comment:
Perhaps use just released `v2.2.190830001` as this version supports transitive build targets, so at least VS2019 users doesn't have to also add this package to their project head